lib/locker/label.rb

```diff
@@ -11,6 +11,7 @@ class Label
       value :taken_at
 
       set :membership
+      set :observer_ids
```

Comment:
Reminder to self before I merge this: this needs to be added to the list in self.delete.